### ISSUE TITLE
Some icons doesn't show up

### ISSUE THREAD

**Issue by Shini31** [bhilburn/powerlevel9k#827]
Hello,

I have an issue with some icons like HOME_ICON or with the clock icon (\Uf017) defined in last prompt prefix:
![icons](https://user-images.githubusercontent.com/333820/39723548-6f828290-5246-11e8-9162-d4c23b358858.png)

For me, all is good in .zshrc file:
![zshrc](https://user-images.githubusercontent.com/333820/39723615-ac1d75de-5246-11e8-9c58-eadf83c2a239.png)

The awesome font is installed.

Does anyone have any idea?

Regards

**Comment by Shini31** [comment]
Fix by editing the 10-powerline-symbols.conf and adding "FontAwesome" as family prefer.

**Comment by Elaniobro** [comment]
Hey @Shini31 how are these icons that you are listing used? Meaning, where do they show up in your terminal, and how do you define these? Thanks!

**Comment by dritter** [comment]
@Elaniobro Not sure if I get you correctly, but these Icons are used in the P9K segments.. So they show up in the prompt..

**Comment by Shini31** [comment]
I had a problem with HOME_ICON and a custom icon that i had defined in P9K segments.
On the first screenshot, you can see how they show up in my terminal.

In fact, I did not put the correct font name in the fontconfig file.

**Comment by Elaniobro** [comment]
@dritter & @Shini31 thank you both for your prompt reply, I confused myself with the tabs I had opened, I did not realize which repo I was in. I found all the documentation for customizing, which has the above and therefore my answer. Apologies if I confused you, but the issue was I confused myself, when I was looking for `powerlevel9k` documentation and not seeing any in the `awesome-terminal-fonts` repo!
